spec/controllers/account/admin/companies_controller_spec.rb

```diff
@@ -110,4 +111,32 @@
       expect(flash[:danger]).to be_present
     end
   end
+
```

Comment:
cover #new_import and #import action test into their own describe block

and move let for file isede of it as there is no need for file in any other specs beside this two